<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Export As:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Export:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>70</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>70</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QFrame" name="export_mode">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>190</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="waypoints_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>97</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Waypoints</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="trajectories_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>97</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Trajectories</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="export_as">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>190</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="csv_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>90</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CSV</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="json_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>97</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>JSON</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cpp_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>97</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>C++</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="java_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>97</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Java</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="python_radio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>97</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Python</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Settings:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QFrame" name="java_settings">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>190</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>180</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Class Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>180</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Package Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="class_name_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>180</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Waypoints</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_name_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>180</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="text_preview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>370</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
